src/sketch/welcomeview.cpp

```diff
@@ -930,22 +926,22 @@ void WelcomeView::getNextBlogImage(int ix, bool blog) {
 }
 
 void WelcomeView::gotBlogImage(QNetworkReply * networkReply) {
-	QNetworkAccessManager * manager = networkReply->manager();
-	if (manager == NULL) return;
+	auto manager = networkReply->manager();
+	if (!manager) return;
 
-	int index = manager->property("index").toInt();
-	bool blog = manager->property("blog").toBool();
+	auto index = manager->property("index").toInt();
+	auto blog = manager->property("blog").toBool();
 
-	int responseCode = networkReply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+	auto responseCode = networkReply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
 	if (responseCode == 200) {
 		QByteArray data(networkReply->readAll());
 		QPixmap pixmap;
 		if (pixmap.loadFromData(data)) {
-			QPixmap scaled = pixmap.scaled(QSize(ImageSpace, ImageSpace), Qt::KeepAspectRatio);
+			auto scaled = pixmap.scaled(QSize(ImageSpace, ImageSpace), Qt::KeepAspectRatio);
```

Comment:
pixmap already is kind of Hungarian notation, auto doesn't improve this. QPixmap is a good guess for scaled() return type, but would you know it for pixmap.mask() ?